ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -22,31 +22,57 @@
 '''
 
 LOOKAHEAD_WPS = 200 # Number of waypoints we will publish. You can change this number
-
+DEFAULT_VELOCITY = 5 # just picked a number for early tests
```

Comment:
Good to get started, but later let's set these as ROS param server values so we can tune them on the fly during testing.

Reply:
I will implement code to set the values in the param server if they don't exist, and to grab them from the param server in send_waypoints.  I'm not sure what the cost is on param_server call - should it only be done every so many iterations? since the func is called so frequently, and changes won't happen that often?

Reply:
Good question. Actually, maybe we should start with it pulling the param server values in the Init so it’s just done once although it may not be tunable on the fly.  I think later we should slow down the waypoint updates anyway instead of on every pose update (50Hz is probably overkill and I can drive faster than they can update in the simulator on my slow machine) and then we can find a good way to pull in updated tuning values then.